thirdparty/libvpx/vpx_dsp/x86/vpx_subpixel_8t_intrin_avx2.c

```diff
@@ -41,7 +41,7 @@ DECLARE_ALIGNED(32, static const uint8_t, filt4_global_avx2[32]) = {
 
 #if defined(__clang__)
 # if __clang_major__ < 3 || (__clang_major__ == 3 && __clang_minor__ <= 3) || \
-    (defined(__APPLE__) && \
+    (!defined(__MACPORTS__) && defined(__APPLE__) && \
```

Comment:
Please add `// -- GODOT start --` and `// -- GODOT end --` comments around your change so that we know that the code was modified, and mention it in `thirdparty/README.md`.